spikesorters/waveclus/waveclus.py

```diff
@@ -220,9 +220,10 @@ def _run(self, recording, output_folder):
 
         if 'win' in sys.platform and sys.platform != 'darwin':
             shell_cmd = '''
+                {disk_move}
                 cd {tmpdir}
                 matlab -nosplash -wait -log -r run_waveclus
-            '''.format(tmpdir=tmpdir)
+            '''.format(disk_move=str(tmpdir)[:2], tmpdir=tmpdir)
```

Comment:
@ferchaure why is this using the first two charachters of tmpdir? To get the disk? For exampl in this case let's say that `str(tmpdir)[:2]` is `D:`: Then does the command `D:` changes directory in Windows?

Reply:
exactly. another annoying windows thing. I found a nice explanation [here].(https://superuser.com/questions/1565202/cannot-change-drive-letter-in-command-prompt-cmd-other-than-c)

Reply:
Awesome! I didn't know :) 

Can I ask you a favor? Could you check for similar cases in the other sorters as well? So we fix it once and for all

Reply:
ok, I'll take a look this week.

Reply:
Thanks!!

Reply:
done